src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -137,6 +137,19 @@ internal Conversion(ConversionKind kind, DeconstructionInfo deconstructionInfo,
             _uncommonData = new DeconstructionUncommonData(deconstructionInfo, nestedConversions);
         }
 
+        public static implicit operator ConversionInfo(Conversion conversion)
```

Comment:
FWIW, I'm not sure this is valuable enough to warrant have two concepts: `Conversion` and `ConversionInfo`.